app/assets/javascripts/frontend/on_ready.js

```diff
@@ -2,7 +2,6 @@
 jQuery(function ($) {
   $('.govspeak').enhanceYoutubeVideoLinks()
 
-  GOVUK.worldLocationFilter.init()
   GOVUK.hideDepartmentChildren.init()
   GOVUK.filterListItems.init()
```

Comment:
I think `filterListItems` is only used on the world index page, so that can go too

Reply:
cheers :)